src/SyncAction/DocsHelper.php

```diff
@@ -17,4 +17,48 @@ public static function mergeHtml(string $html, string $catalogJson, string $mani
 
         return (string) str_replace($searchStr, $newStr, $html);
     }
+
+    /**
+     * @param array<string, mixed> $manifest
+     * @param array<string, mixed> $runResults
+     * @return array<int, array<string, mixed>>
+     */
+    public static function getModelTiming(array $manifest, array $runResults): array
+    {
+        $nodes = $manifest['nodes'];
+        $result = [];
+
+        foreach ($runResults['results'] as $run) {
+            $node = $nodes[$run['unique_id']];
+
+            $executeTimings = array_filter($run['timing'], function ($item) {
+                return $item['name'] === 'execute';
+            });
+            $executeTiming = array_pop($executeTimings);
+
+            $idArray = explode('.', $run['unique_id']);
+
+            // deduplicate
+            $dependsOnNodes = array_filter($node['depends_on']['nodes'], function ($node) {
+                $idArray = explode('.', $node);
+                return array_shift($idArray) === 'model';
+            });
+
+            $dependsOn = [];
+            foreach ($dependsOnNodes as $dependOnNode) {
+                $dependsOn[$dependOnNode] = $dependOnNode;
+            }
```

Comment:
Tohle je proto, ze v tom manifeste z nejakeho dovodu tam muze byt stejna dependence x-krat. 

Napr. v pripade toho beers projektu, tam pro "order_lines" bylo 900x(!) tabulka "beers" jako zavislost 🤷 

![Screenshot 2022-09-20 at 9 58 51](https://user-images.githubusercontent.com/1488015/191201224-f6289ea8-395d-43bf-bf57-f461108fea1e.png)

To jen tak pro zajimavost. Mozna je to bug v DBT, nebo nevim jak to funguje. Kazdopadne pro nase ucely, nam to tam staci jednou :)

Reply:
nestačilo by na to `$dependsOnNodes` zavolat `array_unique()`?

Reply:
🤔 asi jo :)